tests/acceptance/01-InstallJoomlaCept.php

```diff
@@ -9,7 +9,7 @@
 // Load the Step Object Page
 $I = new \AcceptanceTester($scenario);
 
-$I->wantToTest('redCORE installation in Joomla 3');
+$I->wantToTest('Joomla 3 Installation');
```

Comment:
Also please modify JoomlaBrowser debug here:

![screen shot 2015-05-04 at 17 33 24](https://cloud.githubusercontent.com/assets/1375475/7456204/e86d62be-f282-11e4-8903-c8ce745573f7.png)

So it says: 

```
Extension successfully installed from/home/travis/build/redCOMPONENT-COM/redCORE/
Extension successfully installed from: /home/travis/build/redCOMPONENT-COM/redCORE/
```